controllers/helper_test.go

```diff
@@ -1,330 +0,0 @@
-/*
```

Comment:
Why remove this file?

Reply:
this was related to the device plugin one I rebase this PR